src/components/verification-grid/verification-grid.ts

```diff
@@ -1081,25 +1165,82 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
 
   //#endregion
 
-  //#region Templates
+  //#region Prompts
+
+  private hasClassificationTask(): boolean {
+    return this.classificationDecisionElements.length > 0;
+  }
+
+  private hasVerificationTask(): boolean {
+    return this.verificationDecisionElements.length > 0;
+  }
+
+  private mixedTaskPromptTemplate(hasMultipleTiles: boolean, hasSubSelection: boolean): TemplateResult<1> {
+    if (hasSubSelection) {
+      return html`<p>Make a decision about all of the selected</p>`;
+    }
+
+    if (hasMultipleTiles) {
+      return html`<p>Make a decision about all of the tiles</p>`;
+    } else {
+      return html`<p>Make a decision about the shown spectrogram</p>`;
+    }
+  }
+
+  private classificationTaskPromptTemplate(hasMultipleTiles: boolean, hasSubSelection: boolean): TemplateResult<1> {
+    if (hasSubSelection) {
+      return html`<p>Apply labels to selected subjects</p>`;
+    }
+
+    if (hasMultipleTiles) {
+      return html`<p>Apply all relevant labels to all relevant subjects</p>`;
+    } else {
+      return html`<p>Apply all relevant labels to the subject</p>`;
```

Comment:
even more mixed vernacular: you use _classification_ in the help prompt so you should use _classify_ here